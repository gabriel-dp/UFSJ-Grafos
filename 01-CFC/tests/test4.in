A B
B C
B E
B F
C D
C G
D C
D H
E A
E F
F G
G F
G H
H H