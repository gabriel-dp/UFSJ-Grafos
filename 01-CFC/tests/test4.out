{ A B E }
{ C D }
{ F G }
{ H }
